examples/complete/main.tf

```diff
@@ -0,0 +1,10 @@
+
+module "cos_example" {
```

Comment:
One suggestion here is to name it "cos_instance" (or "cos_instance_example") to make it crystal clear it is creating an instance as opposed to a bucket.